### DISCUSSION TITLE
Something went wrong

### DISCUSSION THREAD

**Discussion by pwbank** [iotaledger/firefly#2525] [Help & Support]
![error](https://user-images.githubusercontent.com/27260418/157590936-b79a5091-adf0-4a6a-9239-825c2ee1b78e.JPG)
Hi
I have a problem when I try to mint 2nd NFT
I have try following step your team do on the other topic
what I have done is
1.download and install firefly again
2.log out and make new profile and try to pay

they are still show "Something went wrong" and I have picture of error log to show you

Thank you for your help

**Comment by MiKeF05** [comment]
The "issue" is that you can´t leave less than 1 Mi on an address. So the remainder of the transaction has 750000i (0.75 Mi) which is currently not allowed.
